elasticdl/python/master/k8s_instance_manager.py

```diff
@@ -9,6 +9,22 @@
 _SERVICE_ADDR_SEP = ","
 
 
+def _parse_worker_pod_priority(num_workers, worker_pod_priority):
+    res = {}
+    if isinstance(worker_pod_priority, str) and "high=" in worker_pod_priority:
+        fraction = float(worker_pod_priority.split("=")[1])
```

Comment:
This line may cause ValueError exception if the argument is not in the correct form.
use try to catch exception. If caught, log a warning and use a low priority for all pods.

Reply:
Done